http/vibe/http/server.d

```diff
@@ -585,13 +586,15 @@ struct HTTPServerOption {
 	static enum errorStackTraces          = HTTPServerOptionImpl.errorStackTraces;
 	/// Enable port reuse in `listenTCP()`
 	static enum reusePort                 = HTTPServerOptionImpl.reusePort;
+	/// Enable address reuse in `listenTCP()`
+	static enum reuseAddress              = HTTPServerOptionImpl.reuseAddress;
 
 	/** The default set of options.
 
 		Includes all parsing options, as well as the `errorStackTraces`
 		option if the code is compiled in debug mode.
 	*/
-	static enum defaults = () { debug return HTTPServerOptionImpl.errorStackTraces; else return HTTPServerOptionImpl.none; } ().HTTPServerOption;
+	static enum defaults = () { debug return HTTPServerOptionImpl.errorStackTraces; else return HTTPServerOptionImpl.reuseAddress; } ().HTTPServerOption;
```

Comment:
Uhm, now Vibe.d wouldn't set `reuseAddress` for debug builds ...

Reply:
Oops, fixed.